cmd/earthly/project_cmds.go

```diff
@@ -124,6 +125,17 @@ func (app *earthlyApp) actionProjectRemove(cliCtx *cli.Context) error {
 		return errors.New("project name is required")
 	}
 
+	answer, err := promptInput(cliCtx.Context,
+		"WARNING: you are about to permanently delete this project and all of its associated pipelines, build history and secrets.\n"+
+			"Would you like to continue?\n"+
+			"Type 'y' or 'yes': ")
+	answer = strings.TrimSpace(strings.ToLower(answer))
+
+	if answer != "y" && answer != "yes" {
+		app.console.Printf("Operation aborted.")
+		return nil
+	}
```

Comment:
We generally try not to make anything interactive, unless we have no choice. Reason being that it's harder to work with it in automation.

I think it's ok not to have any prompt here, the same way `rm -rf ~` doesn't present a prompt either.

Reply:
`-f` could be a decent compromise if we want to keep the prompt.

Reply:
If we keep it, can you make the format of the question consistent with other prompts? e.g. I think we had things like `[yN]` in others.

Reply:
I was saying "go ahead and always delete the project". But prompt + option to use -f to skip the prompt is also ok.